src/plugins/git/edges.js

```diff
@@ -140,7 +140,7 @@ function multiLengthDecode(x: $ReadOnlyArray<string>, fail: () => Error) {
 }
 
 export function fromRaw(x: RawAddress): StructuredAddress {
-  function fail() {
+  function fail(_unused_assertion: any) {
```

Comment:
This particular change makes the code less clear. We shouldn’t add an
extra parameter to a function just to avoid a lint suppression.

The best version that I can think of is the original code.

Same comment applies to the other three `fail` functions.